sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -247,6 +247,7 @@ datetimeConstantLiteral
     | DAY_OF_YEAR
     | LOCALTIME
     | LOCALTIMESTAMP
+    | MINUTE_OF_HOUR
```

Comment:
Same as https://github.com/Bit-Quill/opensearch-project-sql/pull/195#discussion_r1052713305

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/196/commits/eb4c99e61b2af5497bcc26e2cc216a021271ec4e